components/03-modules/cart-list-item/CartListItem.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <div class="cart-list-item">
+    <div class="cart-list-item__image">
+      <alpaca-image
+        :src="productImage.dataSrc"
+        :alt="productImage.alt"
+      />
+    </div>
+    <div class="cart-list-item__info">
+      <div class="cart-list-item__data">
+        <alpaca-link
+          :href="href"
+          inverted
+          custom-class="cart-list-item__name"
+        >
+          {{ productName }}
+        </alpaca-link>
+        <div class="cart-list-item__details">
+          <div class="cart-list-item__qty">
+            <div class="cart-list-item__qty-wrapper">
+              <alpaca-input
+                :id="qtyInput.id"
```

Comment:
it would be better to add productId to mock and then use it here `:id="productId"`